src/Workspaces/CSharp/Portable/CodeGeneration/MethodGenerator.cs

```diff
@@ -112,7 +112,7 @@ internal static class MethodGenerator
             var methodDeclaration = SyntaxFactory.MethodDeclaration(
                 attributeLists: GenerateAttributes(method, options, explicitInterfaceSpecifier != null),
                 modifiers: GenerateModifiers(method, destination, options),
-                returnType: method.GenerateReturnTypeSyntax(),
+                returnType: method.GenerateReturnTypeSyntax(method.ReturnNullableAnnotation),
```

Comment:
i don't understand this cahnge.  this looks to be exactly what `.GenerateReturnTypeSyntax()` would do.